spec/chipmunk/bagger/video_game_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "chipmunk/bagger/video_game"
+require "chipmunk/bag"
+
+RSpec.describe Chipmunk::Bagger::VideoGame do
+  let(:external_id) { "12345" }
+  let(:fakeuuid) { "fakeuuid" }
+  let(:fixture_data) { fixture("video_game", "pre-chipmunk") }
```

Comment:
We will need to set up a sample bag that conforms for this test to be meaningful.

Reply:
I added the fixtures we had for digital.

Reply:
Ahh, I see. We will want to have an actual example. It's pretty confusing to have the "video game" be a completely different type.

Reply:
Noah, should I as Lance for an example?  Where do we get the example?

Reply:
I suspect the best approach will be to get a real package and trim it back or adjust portions to fabricate an example that is appropriate for a fixture.

Reply:
I just got a trimmed package that contains nothing private from Lance.  Will send it up with next pr.

Reply:
I guess this is OK. We don't check anything about the profile for any of the types here, just that the basic metadata is added by the bagger subclass. The server will validate the profile and reject any errors. We could add robustness here at any time.